lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -243,6 +244,27 @@ def extract_student(student, features):
     return [extract_student(student, features) for student in may_enroll_and_unenrolled]
 
 
+def get_proctored_exam_results(course_key, features):
+    """
+    Return info about proctored exam results in a course as a dict.
+    """
+    exam_attempts = get_all_exam_attempts(course_key)
+
+    def extract_student(exam_attempt, features):
+        """
+        Build dict containing information about a single student exam_attempt.
+        """
+        proctored_exam_dict = dict(
```

Comment:
You don't have to call the variable a dict -- sort of a backwards Hungarian notation -- you can just call it proctored_exam